### PR TITLE
Rework.

### PR DESCRIPTION
**Pull request by ypid**
https://github.com/ypid/ansible-preseed/blob/rework/CHANGES.rst#v012

Provisioning with preseeding and the late_command script works great. Not sure how many times I already answered those questions of the Debian installer because I was to lazy to setup pressed files :smile:.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -228,10 +228,11 @@ preseed_admin_sshkeys: [ '{{ lookup("pipe", "ssh-add -L | grep ^ssh || cat ~/.ss
 preseed_debian_locale: 'en_US.UTF-8'
 
 
-# .. envvar:: preseed_debian_keyboard_variant
+# .. envvar:: preseed_debian_keyboard_keymap
 #
 # Keyboard layout enabled during installation.
-preseed_debian_keyboard_variant: 'American English'
+# The current keymap layout can be looked up via ``setxkbmap -query``.
+preseed_debian_keyboard_keymap: 'en'
```
Comment (drybjed) [inline defaults/main.yml:235]: Does this affect all of the keyboard internationalization questions? Also, does this work with Debian Wheezy installer?

**Comment by drybjed** [comment]
Very nice work, and thanks for updating the Changelog. :-)

File: defaults/main.yml (hunk 1 above)
Comment (ypid) [inline defaults/main.yml:235]: > Also, does this work with Debian Wheezy installer?

I have not tested Wheezy.

> Does this affect all of the keyboard internationalization questions?

It should. You could also configure the variant but I did not need to do that and for most users the default variant should be ok.

**Comment by ypid** [comment]
No problem, thanks for your hard work and for creating the DebOps project :+1: :+1:

**Pull request merged**